packages/amplify_datastore/test/query_predicate_test.dart

```diff
@@ -474,53 +342,6 @@ void main() {
       final post2 = Post(title: 'post 2', rating: 1, blog: blog1);
       final post3 = Post(title: 'post 3', rating: 1, blog: blog2);
 
-      final postWithNoToMap1 = PostWithNoToMap(
-        title: 'post 1',
-        rating: 1,
-        blog: blog1,
-      );
-      final postWithNoToMap2 = PostWithNoToMap(
-        title: 'post 2',
-        rating: 1,
-        blog: blog1,
-      );
-      final postWithNoToMap3 = PostWithNoToMap(
-        title: 'post 3',
-        rating: 1,
-        blog: blog2,
-      );
-
-      // TODO(Jordan-Nelson): Remove at next major version bump.
-      // can be removed when `getId()` is removed.
-      test('equals (id)', () {
-        final testPredicate = Post.BLOG.eq(blog1.id);
```

Comment:
I think rather than remove this test, and the one immediately after, we can update them to use `modelIdentifier` instead of `id`. I don't see other tests that cover that case.

We should call this out in the v2 migration guide.

```dart
final testPredicate = Post.BLOG.eq(blog1.modelIdentifier);
```

Reply:
Discussed offline. We have other tests that cover this.